nox/sessions.py

```diff
@@ -278,6 +280,7 @@ def install_and_run_script(
             silent=silent,
             success_codes=success_codes,
             external=None,
+            log=log,
```

Comment:
This is a bug fix, it wasn't being passed through.

Reply:
If it was passed, it was ignored before, now it will work as intended. It just affects logging, so it shouldn't break anyone. And this is a very new method, it was added last release.